Microsoft.Toolkit.Uwp.UI/Extensions/TextBoxRegEx/TextBoxRegex.Properties.cs

```diff
@@ -20,7 +20,7 @@ namespace Microsoft.Toolkit.Uwp.UI.Extensions
     /// </summary>
     /// <remarks>
     /// If <see cref="ValidationMode"> is set to Normal then IsValid will be set according to whether the regex is valid.</see>
-    /// If <see cref="ValidationMode"> is set to Forced and the input is not valid the TextBox text will be cleared.</see>
+    /// If <see cref="ValidationMode"> is set to Dynamic, the input will be validated at text changed. If the newest charachter is invalid, only invalid character of the Textbox will be deleted.</see>
```

Comment:
Typo with character. Same in other files (not sure if you need this in every single file btw)